pyvo/dal/scs.py

```diff
@@ -413,7 +413,7 @@ def execute(self):
         return SCSResults(self.execute_votable(), url=self.queryurl, session=self._session)
 
 
-class SCSResults(DALResults, DatalinkResultsMixin):
```

Comment:
This new inheritance ordering makes sense as it's more consistent with the other *Results classes, but I'm idly curious whether it was a style change or you found you needed this to achieve the desired behavior.

Reply:
Ah never mind.  I see this is already mentioned in #292.